test/lib/independence/index.test.js

```diff
@@ -0,0 +1,105 @@
+const chai = require('chai')
+const sinon = require('sinon')
+const spies = require('chai-spies')
+const axios = require('axios')
+const proxyquire = require('proxyquire')
+const Promise = require('bluebird')
+
+chai.use(spies)
+
+const requestMock = {
+  send: () => Promise.resolve(mockResponse)
+}
+const mockResponse = {
+  data: { field1: 'value1' },
+  status: 200
+}
+let mockAdapter
+let pendingRequests
+const IndependentRequest = chai.spy(data => {
+  requestMock.data = data
+  return requestMock
+})
+
+const { makeIndependent } = proxyquire('../../../lib/independence', {
+  './independent_request': {
+    IndependentRequest
+  }
+})
+
+const expect = chai.expect
+
+function retryedPendingRequests () {
```

Comment:
I think a better name would be with except format such as `exceptPendingRequestsToBeenRetryed`. This will infer that the function is doing some complex assertion and also would be more clear what the function is checking.